_docs/development.md

```diff
@@ -69,6 +79,7 @@ pinned dependencies for RapidPro can be found in ```pip-freeze.txt```. You can
 build the needed environment as follows (from the root rapidpro directory):
 
 {% highlight bash %}
+$ sudo apt-get install python-virtualenv postgresql-server-dev-9.3 python-dev ncurses-dev
 $ virtualenv env
```

Comment:
the build was failing when it couldn’t find the libraries.   is there perhaps an issue on the pip-freeze side of things?
Cheers,
-carl



> On Jan 29, 2015, at 1:45 PM, Nic Pottier <notifications@github.com> wrote:
> 
> In _docs/development.md <https://github.com/rapidpro/rapidpro/pull/124#discussion_r23791568>:
> 
> > @@ -69,6 +79,7 @@ pinned dependencies for RapidPro can be found in ```pip-freeze.txt```. You can
> >  build the needed environment as follows (from the root rapidpro directory):
> >  
> >  {% highlight bash %}
> > +$ sudo apt-get install python-virtualenv postgresql-server-dev-9.3 python-dev ncurses-dev
> >  $ virtualenv env
> This is installing into the system python libraries, which we probably don't want to recommend. (the one exception being virtualenv, the others are part of pip-freeze and shouldn't be required)
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/rapidpro/rapidpro/pull/124/files#r23791568>.
>